src/app/modules/authController.js

```diff
@@ -14,16 +30,42 @@ class UserController {
       password: hashPassword,
     };
     const result = await UserServices.create(authData);
-    const { userId } = result.dataValues;
-    const token = jwt.createToken(userId);
 
+    const uID = req.body.username;
+    const getId = await UserServices.findUserid(uID);
```

Comment:
Don’t need to find the **userId** by making the request. Destruct the **result**

Reply:
okay thanks